spec/services/usps_in_person_proofing/enrollment_helper_spec.rb

```diff
@@ -58,25 +67,69 @@
         expect(enrollment.current_address_matches_id).to eq(current_address_matches_id)
       end
 
-      it 'creates usps enrollment' do
-        proofer = UspsInPersonProofing::Mock::Proofer.new
-        mock = double
-
-        expect(UspsInPersonProofing::Proofer).to receive(:new).and_return(mock)
-        expect(mock).to receive(:request_enroll) do |applicant|
-          expect(applicant.first_name).to eq(Idp::Constants::MOCK_IDV_APPLICANT[:first_name])
-          expect(applicant.last_name).to eq(Idp::Constants::MOCK_IDV_APPLICANT[:last_name])
-          expect(applicant.address).to eq(Idp::Constants::MOCK_IDV_APPLICANT[:address1])
-          expect(applicant.city).to eq(Idp::Constants::MOCK_IDV_APPLICANT[:city])
-          expect(applicant.state).to eq(Idp::Constants::MOCK_IDV_APPLICANT[:state])
-          expect(applicant.zip_code).to eq(Idp::Constants::MOCK_IDV_APPLICANT[:zipcode])
-          expect(applicant.email).to eq('no-reply@login.gov')
-          expect(applicant.unique_id).to eq(enrollment.unique_id)
-
-          proofer.request_enroll(applicant)
+      context 'transliteration disabled' do
+        let(:usps_ipp_transliteration_enabled) { false }
+
+        it 'creates usps enrollment without using transliteration' do
+          proofer = UspsInPersonProofing::Mock::Proofer.new
+          mock = double
+
+          expect(transliterator).not_to receive(:transliterate)
+          expect(UspsInPersonProofing::Proofer).to receive(:new).and_return(mock)
+          expect(mock).to receive(:request_enroll) do |applicant|
+            expect(applicant.first_name).to eq(Idp::Constants::MOCK_IDV_APPLICANT[:first_name])
+            expect(applicant.last_name).to eq(Idp::Constants::MOCK_IDV_APPLICANT[:last_name])
+            expect(applicant.address).to eq(Idp::Constants::MOCK_IDV_APPLICANT[:address1])
+            expect(applicant.city).to eq(Idp::Constants::MOCK_IDV_APPLICANT[:city])
+            expect(applicant.state).to eq(Idp::Constants::MOCK_IDV_APPLICANT[:state])
+            expect(applicant.zip_code).to eq(Idp::Constants::MOCK_IDV_APPLICANT[:zipcode])
+            expect(applicant.email).to eq('no-reply@login.gov')
+            expect(applicant.unique_id).to eq(enrollment.unique_id)
+
+            proofer.request_enroll(applicant)
```

Comment:
we can avoid stubbing `new` here as well, and even avoid having separate `proofer` from `mock` instances. I suspect we could even drop the explicit `request_enroll` call and just throw on `.and_call_original` if `mock_proofer` was an instance of `UspsInPersonProofing::Mock::Proofer`


```suggestion
          mock_proofer = double('UspsInPersonProofing::Proofer')
          expect(subject).to receive(:usps_proofer).and_return(mock_proofer)

          expect(transliterator).not_to receive(:transliterate)

          expect(mock_proofer).to receive(:request_enroll) do |applicant|
            expect(applicant.first_name).to eq(Idp::Constants::MOCK_IDV_APPLICANT[:first_name])
            expect(applicant.last_name).to eq(Idp::Constants::MOCK_IDV_APPLICANT[:last_name])
            expect(applicant.address).to eq(Idp::Constants::MOCK_IDV_APPLICANT[:address1])
            expect(applicant.city).to eq(Idp::Constants::MOCK_IDV_APPLICANT[:city])
            expect(applicant.state).to eq(Idp::Constants::MOCK_IDV_APPLICANT[:state])
            expect(applicant.zip_code).to eq(Idp::Constants::MOCK_IDV_APPLICANT[:zipcode])
            expect(applicant.email).to eq('no-reply@login.gov')
            expect(applicant.unique_id).to eq(enrollment.unique_id)

            UspsInPersonProofing::Mock::Proofer.new.request_enroll(applicant)
```

Reply:
I incorporated these changes, but included additional lines not listed in the suggestion.